### ISSUE TITLE
RC Test: Balloons and Static Electricity 1.4.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#135]
@emily-phet @terracoda @ariel-phet, Balloons and Static Electricity 1.4.0-rc.1 is ready for RC testing.

**[Link to sim](https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.4.0-rc.1/phet/balloons-and-static-electricity_all_phet_debug.html)**

**[Link to iFrame](https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.4.0-rc.1/phet/balloons-and-static-electricity_all_iframe_phet.html)**

**[Test Matrix](https://docs.google.com/spreadsheets/d/153iOnhpi5IHdqFp3JBC9QDOmToPjhf4idY9YXQkqtac/edit#gid=646295854)**

**Before beginning**, please familiarize yourself with how a screen reader works. Here is a page with information about PhET's supported screen readers and documentation about how to use them:
[Screen reader intro](https://www.colorado.edu/physics/phet/dev/html/jg-tests/reader-intro.html)

The simulation to test has labels and/or descriptions. They can be "seen" here:
**[Link to a11y view](http://www.colorado.edu/physics/phet/dev/html/{{REPO}}/phet/1.X.X-rc.1/{{REPO}}_a11y_view.html)**

This view shows what is available to a screen reader user. The sim is on the left side in an iframe, while the right side has all of the accessible descriptions. Beneath the sim are real-time alerts that will be announced by the screen reader. Please make sure that the descriptions on the right accurately describe the simulation at all times.

PhET supports the following platforms for accessibility so please test these:
*(These should be in the test matrix)*
- [ ] JAWS with latest Windows, latest Firefox
- [ ] NVDA with latest Windows, latest Firefox
- [ ] VoiceOver with latest macOS, latest Safari

**Please also verify**
- [ ] stringTest=double (all strings doubled)
- [ ] stringTest=long (exceptionally long strings)
- [ ] stringTest=X (short strings)
- [ ] stringTest=rtl (right-to-left)
- [ ] stringTest=xss (test passes if sim does not redirect, OK if sim crashes or fails to fully start)
- [ ] showPointerAreas (touchArea=red, mouseArea=blue)
- [ ] Full screen test
- [ ] Screenshot test
- [ ] Does AR need to be notified for any LOL updates?

Accessibility strings that are not visible should not be translatable yet, do not worry if these do not change with string tests.

If any new issues are found, please note them in https://github.com/phetsims/balloons-and-static-electricity/issues and reference this issue. 

**Critical screen reader information**
  We are tracking known AT bugs in https://docs.google.com/document/d/1518zv6F0odExFsodShZxwTmWNQk3civuNwtQUR-rEBs/. If
  you notice a new bug with AT behavior, please check against this list first.
 
**(Other potentially useful items)**  

Sim-specific query parameters useful for testing:  
• `?showGrid` - Shows described areas in balloons-and-static-electricity, balloon location description will change depending on which cell it is in.

@ariel-phet please assign and prioritize

**Comment by ariel-phet** [comment]
@lmulhall-phet as you can tell from the liberal use of emojis by @terracoda this is kind of a big deal. Current top testing priority

**Comment by JRomero0613** [comment]
I'm not sure what happened but the sim link now directs to the testing matrix google sheet, is someone able to fix this so that the link directs to the sim?

**Comment by lmulhall-phet** [comment]
@JRomero0613, you can edit the issue to fix things like this in the future.

**Comment by lmulhall-phet** [comment]
@jessegreenberg, QA is done!

**Comment by terracoda** [comment]
@lmulhall-phet, are you serious? That was super quick! This is fabulous news! Thanks for all the hard work! 

@emily-phet , @jessegreenberg, @ariel-phet, QA is done, and we just have one issue!
